### PR TITLE
Fixes #37156 - Pin victory-core to 36.8.z

### PR DESCRIPTION
**Pull request by sjha4**
Pin victory-core to less than 36.9.0 which breaks react-charts. See https://github.com/patternfly/patternfly-react/issues/10064

### CODE REVIEW

**Comment by jeremylenz** [comment]
Is Foreman the correct place for it? We only use it in Katello..

**Comment by sjha4** [comment]
> Is Foreman the correct place for it? We only use it in Katello..

I thought REX uses charts too..Let me check..

So I found it in these places in foreman project:

https://github.com/search?q=org%3Atheforeman%20%40patternfly%2Freact-charts&type=code

**Comment by sjha4** [comment]
@evgeni Does this look good?

File: package.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
     "webpack": "^5.75.0",
     "webpack-bundle-analyzer": "^4.5.0",
     "webpack-cli": "^5.0.1",
-    "webpack-stats-plugin": "^1.0.3"
+    "webpack-stats-plugin": "^1.0.3",
+    "victory-core": "~36.8.6"
```
Comment (MariaAga) [inline package.json]: can you put it in alphabetical order?

File: package-exclude.json [hunk 2]
```diff
@@ -28,7 +28,8 @@
         "surge",
         "webpack-bundle-analyzer",
         "tabbable",
-        "@adobe/css-tools"
+        "@adobe/css-tools",
+        "victory-core"
```
Comment (MariaAga) [inline package-exclude.json:32]: Why is it excluded?

Comment (sjha4) [reply package-exclude.json:32]: I am not seeing this packaged today so not sure we want to package it..

Comment (sjha4) [reply package-exclude.json:32]: @evgeni Is this supposed to exclude only dev dependencies?

Comment (ekohl) [reply package-exclude.json:32]: I'm unsure about this change, but the list must remain sorted.

**Review by evgeni — APPROVED** [review]
the lists being sorted would be cool

**Pull request merged**